src/ReactiveDomain.UI.Testing/TestVisibilityHelpers.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Reflection;
+
+namespace ReactiveDomain.UI.Testing
+{
+    public static class TestVisibilityExtensions
+    {
+        public static T CurrentValue<T>(this IObservable<T> readModelProperty)
```

Comment:
Why make this an extension method instead of adding it directly to `ReadModelProperty`? That would let us avoid having to use reflection and would presumably run faster.

Reply:
It would also lead to incorrect usage, this clearly indicates this should only be used in testing via the namespace.
Not doing more of this led to a bunch of what I just cleaned up. (@jageall can expound at length)